plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -436,6 +456,11 @@ static int deactivate(struct ffmpeg_muxer *stream, int code)
 		os_atomic_set_bool(&stream->active, false);
 		os_atomic_set_bool(&stream->sent_headers, false);
 
+		service = obs_output_get_service(stream->output);
+		if (!service)
```

Comment:
You can't just return here. The rest of the function still needs to be executed.

Reply:
Good point. Fixed.